cmd/foreach/foreach.go

```diff
@@ -65,7 +80,9 @@ marks that no further options should be interpreted by turbolift.`,
 		Args: cobra.MinimumNArgs(1),
 	}
 
-	cmd.Flags().StringVar(&repoFile, "repos", "repos.txt", "A file containing a list of repositories to clone.")
+	cmd.Flags().StringVar(&repoFile, "repos", "", "A file containing a list of repositories to clone.")
```

Comment:
I'm not sure this is needed - this mutex between repos.txt and successful/failed seems complicated, should we just ignore the repoFile if --successful or --failed is set?

Reply:
yeah perhaps unnecessary, I just thought that if someone is a bit confused, they could do `turbolift foreach --failed --repos repos.txt -- some command` and not realise what is going on
happy to change it back though

Reply:
Following discussion, I've updated this to still check that only one flag has been set, but not do the convoluted logic of resetting the default value. My example above is very unlikely and also covered by the readme instructions